src/System/Router/Boot.tsx

```diff
@@ -87,9 +88,11 @@ export const Boot = track(undefined, {
                   >
                     <ToastsProvider>
                       <StickyProvider>
-                        <FocusVisible />
-                        <SiftContainer />
-                        {children}
+                        <AuthIntentProvider>
+                          <FocusVisible />
+                          <SiftContainer />
+                          {children}
+                        </AuthIntentProvider>
```

Comment:
I do kinda wanna write a little compose providers function to clean this up.

Reply:
If you can pull it off i'll send ya $5. We've all tried...